sections/header.liquid

```diff
@@ -92,8 +93,8 @@
                       </span>
                     </a>
                     {% if childlink.items_count > 0 %}
-                      <input type="checkbox" id="header__nav-grandchild-{{ forloop.index }}">
-                      <label for="header__nav-grandchild-{{ forloop.index }}">
+                      <input type="checkbox" id="header__nav-{{ parent_index }}-grandchild-{{ forloop.index }}">
```

Comment:
Do we need both `parent_index` and `forloop.index`? Aren't they the same?

Reply:
`parent_index` is assigned in `for link in section.settings.menu.items` while there can be next forloop for childlinks

Reply:
So, `parent_index` represents an index of the first loop to properly connect labels to inputs for 3rd level menu items

Reply:
Ahh, gotcha!